packages/node_modules/pouchdb-adapter-indexeddb/src/allDocs.js

```diff
@@ -37,11 +37,16 @@ function createKeyRange(start, end, inclusiveEnd, key, descending) {
 function handleKeyRangeError(opts, metadata, err, callback) {
   if (err.name === "DataError" && err.code === 0) {
     // data error, start is less than end
-    return callback(null, {
+    var returnVal = {
       total_rows: metadata.doc_count,
       offset: opts.skip,
       rows: []
-    });
+    };
+    /* istanbul ignore if */
+    if (opts.update_seq) {
+      returnVal.update_seq = metadata.seq;
+    }
+    callback(null, returnVal);
```

Comment:
Hey @dharders you missed a `return` there. When running the integration test I noticed this. (Sadly it's not so straightforward to do so with the `indexeddb `adapter as there is no query parameter. You need to manualy change some stuff)

Reply:
Yep, you're right. Fixed in #6864 (I must have missed this after rebase manual changes I made)